.github/workflows/ci-cd.yaml

```
@@ -53,12 +53,18 @@ jobs:
           DOCKER_HUB_REPOSITORY: ${{ secrets.DOCKER_HUB_REPOSITORY }}
         run: |
           echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin
+          if [ "${{ github.base_ref }}" = "refs/heads/develop" ]; then
+                DOCKER_TAG="develop"
+          elif [ "${{ github.base_ref }}" = "refs/heads/main" ]; then
+                DOCKER_TAG="main"
+          fi
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)